helper/src/components/deployTab.js

```diff
@@ -220,9 +220,13 @@ export default function DeployTab({ defaults, updateFn, tabValues, invalidArray,
     post_deploycmd.replaceAll('"', '\\"') +
     `\n"${deploy.selectedTemplate === "local" ? ' --file ./postdeploy/scripts/postdeploy.sh --file ./postdeploy/helm/Az-CertManagerIssuer-0.3.0.tgz --file ./postdeploy/k8smanifests/networkpolicy-deny-all.yml --file ./helper/src/dependencies.json' : ''}`
 
+  const networkWatcher = net.nsg && net.nsgFlowLogs !== defaults.net.nsgFlowLogs ?
+    `# Create Network Watcher Resource Group If It Doesn't Exist\n ` +
```

Comment:
if you can get rid of this one character of whitespace - then we're good to approve/merge @MattLeach25

Reply:
Resolved.